### PR TITLE
Fixes (and slightly rewalances) the Robotic Cradle

### LINKED ISSUE #13561: robotic cradle lacks a sprite for when it's operational and occupied
### Description

the robotic cradle doesn't have a consistent/finished sprite for when it's working with power and operating on an occupant

### Test Merges

Server-Side Code Modifications: https://tgstation13.org/parsed-logs/sybil/server_side_modifications.dm

Round ID: 22357
BYOND Version: 515.1608
Server revision compiled on: 
Master commit: [8f501b9154e3f24694fef25423e6a03d52ff89a2](https://github.com/tgstation/TerraGov-Marine-Corps//commit/8f501b9154e3f24694fef25423e6a03d52ff89a2)
The following pull requests are currently test merged:
[#13409: 'The Gun Sound Overhaul 2.0 + More Weapons to EORD (PART 1)' by XSlayer300 at commit 9bae97f5a6](https://github.com/tgstation/TerraGov-Marine-Corps//pull/13409)
[#13504: 'TTS updates: Xeno TTS, new robo/vending voice version, pitch settings, etc' by TiviPlus at commit e537749333](https://github.com/tgstation/TerraGov-Marine-Corps//pull/13504)
[#13426: 'Adds a HUD text timer for both hive collapse and nuke' by Arturlang at commit e26d1ba018](https://github.com/tgstation/TerraGov-Marine-Corps//pull/13426)
[#13515: 'Adds 2 new tadpole models' by XANAv2 at commit d98aa86a61](https://github.com/tgstation/TerraGov-Marine-Corps//pull/13515)
[#13014: 'Queen bulwark' by TiviPlus at commit 75f6e04626](https://github.com/tgstation/TerraGov-Marine-Corps//pull/13014)

Local commit: 175171b556b7acb29624232806cc0e5dc7b890a0
Server tools version: 5.13.2

### Reproduction Steps

1. be a robot
2. gain integrity loss from psydrain or cocoon
3. be revived through defibrillation
4. go shipside to fix integrity damage
5. enter into the robotic cradle
6. the sprite will change once the occupant it in, from the cradle sprite to the animated sprite for a dropship chair being collapsed

### Screenshots

_No response_

### PR DESCRIPTION
**Pull request by efzapa**
## About The Pull Request

- Fixes sprite code so that an active Robotic Cradle should no longer ERP the floor
- Also replaces the no_icon icon in objects.dmi with proper icon instead of the folding chair we had
- Now removes the training limit for people to drag you into cradles (there is still a do_after of 1 second) so it acts more like a cryo tube
- fixes #13561 

## Why It's Good For The Game

Weird inconsistent code/sprites are generally bad

## Changelog
:cl:
balance: Robotic Cradle skill limit completely removed.
fix: Robotic Cradle sprite fixed.
/:cl:

### CODE REVIEW

File: code/game/objects/machinery/robotic_cradle.dm [hunk 1]
```diff
@@ -264,7 +246,7 @@
 
 	M.forceMove(src)
 	occupant = M
-	icon_state = "pod_1"
+	icon_state = "borgcharger1"
```
Comment (Xander3359) [inline code/game/objects/machinery/robotic_cradle.dm:249]: Just delete this completely.

File: code/game/objects/machinery/robotic_cradle.dm [hunk 2]
```diff
@@ -127,8 +123,8 @@
 			reason = "Reason for discharge: Destruction of linked CRADLE Engineering System. Alerting security advised."
 	radio.talk_into(src, "<b>Patient: [occupant] has been released from [src] at: [get_area(src)]. [reason]</b>", RADIO_CHANNEL_MEDICAL)
 	occupant = null
-	update_icon()
 	stop_processing()
+	update_icon()
```
Comment (Xander3359) [inline code/game/objects/machinery/robotic_cradle.dm:127]: You don't need to do this since you fixed update_icon_state()

**Review by Xander3359 — CHANGES_REQUESTED** [review]
Also on line 152 delete ``icon_state = "pod_0"``

File: code/game/objects/machinery/robotic_cradle.dm (hunk 2 above)
Comment (efzapa) [reply code/game/objects/machinery/robotic_cradle.dm:127]: tested, still need this otherwise there is nothing to call the icon change when a synth leaves the cradle

File: code/game/objects/machinery/robotic_cradle.dm (hunk 1 above)
Comment (efzapa) [reply code/game/objects/machinery/robotic_cradle.dm:248]: tested, need this apperently

File: code/game/objects/machinery/robotic_cradle.dm [hunk 3]
```diff
@@ -71,6 +65,8 @@
 	if(!repairing)
 		return
 
+	update_icon()
```
Comment (Lumipharon) [inline code/game/objects/machinery/robotic_cradle.dm:68]: why is this needed every process tick? You have 3 specific things that can change the icon state, you should be calling update icon specifically where those change.

**Review by Lumipharon — CHANGES_REQUESTED** [review]

File: code/game/objects/machinery/robotic_cradle.dm (hunk 3 above)
Comment (efzapa) [reply code/game/objects/machinery/robotic_cradle.dm]: fixed

Comment (Lumipharon) [reply code/game/objects/machinery/robotic_cradle.dm]: Don't self resolve please. Are you sure you've actually done this correctly as opposed to just removing this?

Comment (efzapa) [reply code/game/objects/machinery/robotic_cradle.dm]: well unless you know something i don't the only two things updating the icon should be update_icon_state and go_out, (plus the other checks like occupant and whatnot but those are conditional)

Comment (Lumipharon) [reply code/game/objects/machinery/robotic_cradle.dm]: I only see the icon being update in go_out. What about going in?

Comment (efzapa) [reply code/game/objects/machinery/robotic_cradle.dm]: > I only see the icon being update in go_out. What about going in?

that would be move_inside_wrapper which deals with the occupant var, which in turn is involved with update_icon_state

Comment (Lumipharon) [reply code/game/objects/machinery/robotic_cradle.dm]: ? Unless I am blind, that doesn't call update icon

Comment (efzapa) [reply code/game/objects/machinery/robotic_cradle.dm]: bruh

actually not even sure if it's even move_inside_wrapper since i didn't touch it
i just know the fix works and thats why i made the PR, if something is in fact broken then tell me

Comment (Lumipharon) [reply code/game/objects/machinery/robotic_cradle.dm]: Its not working. You removed it like I asked from processing but you didn't add it anywhere else it needs to be.

Comment (efzapa) [reply code/game/objects/machinery/robotic_cradle.dm]: it does not need to be anywhere
1. the code works, why would i bother putting that anywhere else
2. Capture this handles the icon_change going out

Comment (Lumipharon) [reply code/game/objects/machinery/robotic_cradle.dm]: Bruh, I tested it, jump in, the icon doesn't update. Finishes the repair cycle and boots you out, it updates then.
So it never changes from the inactive state

Comment (efzapa) [reply code/game/objects/machinery/robotic_cradle.dm]: > Bruh, I tested it, jump in, the icon doesn't update. Finishes the repair cycle and boots you out, it updates then. So it never changes from the inactive state

guh, turns out synth can't even drag themself into a robotic cradle which is why i apparently didn't see this
